templates/CONTRIBUTING.md

```diff
@@ -121,11 +92,39 @@ We don't follow any other convention, but if you want to use one, we recommend [
 ### GitHub Pull Requests
 
 Some notes on GitHub PRs:
+
 - [Convert your PR as a draft](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request) if your changes are a work in progress: no one will review it until you pass your PR as ready for review.<br>
   The draft PR can be very useful if you want to show that you are working on something and make your work visible.
-- The branch related to the PR must be **up-to-date with `master`** before merging. You need to [rebase your branch](https://gist.github.com/curquiza/5f7ce615f85331f083cd467fc4e19398) if it is not.
+- The branch related to the PR must be **up-to-date with `master`** before merging. Fortunately, this project [integrates a bot](https://github.com/meilisearch/integration-guides/blob/master/guides/bors.md) to automatically enforce this requirement without the PR author having to do it manually.
 - All PRs must be reviewed and approved by at least one maintainer.
-- All PRs have to be **squashed and merged**.
 - The PR title should be accurate and descriptive of the changes. The title of the PR will be indeed automatically added to the next [release changelogs](https://github.com/meilisearch/XXX/releases/).
 
+## Release Process (for Admin only)
+
+MeiliSearch tools follow the [Semantic Versioning Convention](https://semver.org/).
+
+### Automation to Rebase and Merge the PRs
+
+This project integrates a bot that helps us manage pull requests merging.<br>
+_[Read more about this](https://github.com/meilisearch/integration-guides/blob/master/guides/bors.md)._
+
+### Automated Changelogs
+
+This project integrates a tool to create automated changelogs.<br>
+_[Read more about this](https://github.com/meilisearch/integration-guides/blob/master/guides/release-drafter.md)._
```

Comment:
This throws 404, I guess it is normal since #49 isn't merged, but URL seems to be well written ;)

Reply:
I'm waiting for your review to merge it 😉

Reply:
I was reviewing it as you wrote this 😂